### PR TITLE
Add Transmission package

### PR DESCRIPTION
**Pull request by beliefless-dogmasless**
I'm on arm, it works well.

### CODE REVIEW

File: packages/transmission.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  
+  depends_on 'libevent'
+  depends_on 'openssl'
+  depends_on 'miniupnpc'
+  depends_on 'curl'
+  depends_on 'zlibpkg'
+  depends_on 'glib'
+  depends_on 'rtmpdump'
+  depends_on 'ctorrent'
+  depends_on 'intltool'
```
Comment (uberhacker) [inline packages/transmission.rb]: Build only dependency?

File: packages/transmission.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on 'gettext'
+  depends_on 'pkgconfig'
```
Comment (uberhacker) [inline packages/transmission.rb]: Build only dependency?

File: packages/transmission.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on 'libevent'
+  depends_on 'openssl'
+  depends_on 'miniupnpc'
+  depends_on 'curl'
+  depends_on 'zlibpkg'
+  depends_on 'glib'
+  depends_on 'rtmpdump'
+  depends_on 'ctorrent'
+  depends_on 'intltool'
+  depends_on 'gettext'
```
Comment (uberhacker) [inline packages/transmission.rb]: Build only dependency?

File: packages/transmission.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  })
+  
+  depends_on 'libevent'
+  depends_on 'openssl'
+  depends_on 'miniupnpc'
+  depends_on 'curl'
+  depends_on 'zlibpkg'
+  depends_on 'glib'
+  depends_on 'rtmpdump'
+  depends_on 'ctorrent'
```
Comment (uberhacker) [inline packages/transmission.rb]: Is this really needed?

Comment (beliefless-dogmasless) [reply packages/transmission.rb]: I copy paste, the gentoo...:

https://gitweb.gentoo.org/repo/gentoo.git/diff/net-p2p/transmission/transmission-2.93.ebuild?id=4c23097d5cc80886f0ccbc5061caa5d10d08406f

**Comment by beliefless-dogmasless** [comment]
I removed gettext, intltool and pkgconfig.

**Pull request merged**